scripts/manual_validation_results.qmd

```diff
@@ -0,0 +1,917 @@
+    n = n()
+  )
+
+# CALCULATION FOR ONLY DISCREPANT DATES 
+
+discrepancy_data |>
+  # Filter the data
+  filter(
+    overall_recruitment_status_reg1 == "Completed",
+    overall_recruitment_status_reg2 == "Completed",
+    !is.na(completion_month_year_reg1),
+    completion_month_year_reg1 != completion_month_year_reg2
+  ) |>
+  # Calculate the month difference
+  mutate(
+    month_diff = abs(interval(completion_month_year_reg1, completion_month_year_reg2) %/% months(1))
+  ) |>
+  # Summarize descriptive statistics
+  summarise(
+    mean_month_diff = mean(month_diff),
+    median_month_diff = median(month_diff),
+    sd_month_diff = sd(month_diff),
+    min_month_diff = min(month_diff),
+    max_month_diff = max(month_diff),
+    n = n()
+  )
+```
+
+#### Percentage of cases where EUCTR completion date was before/after/same
+
+```{r message = FALSE, warning = FALSE}
+
+#overall
+discrepancy_data |>
+  filter(overall_recruitment_status_reg1 == "Completed",
+         overall_recruitment_status_reg2 == "Completed",
+         !is.na(completion_month_year_reg1)) |>
+  mutate(date_comparison = case_when(
+    completion_month_year_reg1 < completion_month_year_reg2 ~ "Earlier",
+    completion_month_year_reg1 > completion_month_year_reg2 ~ "Later",
+    TRUE ~ "Same")) |>
+  count(date_comparison) |>
+  mutate(percentage = n / sum(n) * 100)
+
+#ClinicalTrials.gov
+discrepancy_data |>
+  filter(registry2 == "ClinicalTrials.gov") |>
+  filter(overall_recruitment_status_reg1 == "Completed",
+         overall_recruitment_status_reg2 == "Completed",
+         !is.na(completion_month_year_reg1)) |>
+  mutate(date_comparison = case_when(
+    completion_month_year_reg1 < completion_month_year_reg2 ~ "Earlier",
+    completion_month_year_reg1 > completion_month_year_reg2 ~ "Later",
+    TRUE ~ "Same")) |>
+  count(date_comparison) |>
+  mutate(percentage = n / sum(n) * 100)
+
+#DRKS
+discrepancy_data |>
+  filter(registry2 == "DRKS") |>
+  filter(overall_recruitment_status_reg1 == "Completed",
+         overall_recruitment_status_reg2 == "Completed",
+         !is.na(completion_month_year_reg1)) |>
+  mutate(date_comparison = case_when(
+    completion_month_year_reg1 < completion_month_year_reg2 ~ "Earlier",
+    completion_month_year_reg1 > completion_month_year_reg2 ~ "Later",
+    TRUE ~ "Same")) |>
+  count(date_comparison) |>
+  mutate(percentage = n / sum(n) * 100)
+```
+
+### Sensitivity analysis
+
+This is a sensitivity analysis of completion date, where completion dates in the EUCTR were solely taken from the EU member state protocol (completion dates on EUCTR Results pages were disregarded)."
+
+#### Count of observations included in the analysis
+
+```{r message = FALSE, warning = FALSE}
+
+discrepancy_data |>
+  filter(overall_recruitment_status_reg1 == "Completed" & overall_recruitment_status_reg2 == "Completed") |>
+  count(is.na(completion_month_year_protocol) | completion_month_year_protocol == "")
```

Comment:
@elisabascunan do you expect some to be "" instead of NA? I can't see any am not sure why they would be there, but please check and update accordingly

Reply:
I also think this alternative be a bit clearer (otherwise the results show the number of NA and TRUE, where TRUE is actually the number of NAs, which is confusing):

```
discrepancy_data |>
    filter(overall_recruitment_status_reg1 == "Completed" & overall_recruitment_status_reg2 == "Completed") |>
    count(!is.na(completion_month_year_protocol))

discrepancy_data |>
    filter(overall_recruitment_status_reg1 == "Completed" & overall_recruitment_status_reg2 == "Completed") |>
    count(!is.na(completion_month_year_reg2))
```